### PR TITLE
namespace fix; form action path fix

### PR DESCRIPTION
**Pull request by laserchicken**
This change fixes `NameError (uninitialized constant Rizzo::SailthruController::SailthruService)`. Calling SailthruSevice from Rizzo::SailthruController was throwing NameError due to improper nesting (see. http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#nesting). I decided to use module nesting convention instead and it fixed the problem.

### CODE REVIEW

File: app/views/layouts/partials/snippets/_footer_sitemap.html.haml [hunk 1]
```diff
@@ -12,7 +12,7 @@
         %strong
           20% discount
         on your next guidebook purchase
-      %form.newsletter-footer.js-newsletter-footer(action='//www.lonelyplanet.com/rizzo_sailthru' method='post')
+      %form.newsletter-footer.js-newsletter-footer(action='/rizzo_sailthru' method='post')
```
Comment (robuye) [inline app/views/layouts/partials/snippets/_footer_sitemap.html.haml]: what's the difference between this and previous version?

File: app/controllers/rizzo/sailthru_controller.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Rizzo::SailthruController < ActionController::Base
 
   def save
-    render json: {}, status: SailthruService.register(form_params)
+    render json: {}, status: Rizzo::SailthruService.register(form_params)
```
Comment (robuye) [inline app/controllers/rizzo/sailthru_controller.rb]: Why is this necessary? Won't rails lookup `SailthruService` within `Rizzo` namespace anyways?

File: app/views/layouts/partials/snippets/_footer_sitemap.html.haml (hunk 1 above)
Comment (laserchicken) [inline app/views/layouts/partials/snippets/_footer_sitemap.html.haml]: no difference really, just a shorter version

File: app/controllers/rizzo/sailthru_controller.rb (hunk 2 above)
Comment (laserchicken) [inline app/controllers/rizzo/sailthru_controller.rb]: sometimes I get `NameError (uninitialized constant Rizzo::SailthruController::SailthruService)`, after adding this rails lookup seems to work properly

**Comment by laserchicken** [comment]
failing build is not related to this change and is fixed https://github.com/lonelyplanet/rizzo/commit/e99ebb130850e39f3f3f147ee7d2040842939b32

**Comment by tkali** [comment]
:+1:

File: app/views/layouts/partials/snippets/_footer_sitemap.html.haml (hunk 1 above)
Comment (nidalaa) [inline app/views/layouts/partials/snippets/_footer_sitemap.html.haml]: @laserchicken  what about apps with different domain? `shop.lonelyplanet.com` or `auth.lonelyplanet.com`? Would it work?

Comment (nidalaa) [inline app/views/layouts/partials/snippets/_footer_sitemap.html.haml]: ooops, too late... anyway I summon you @laserchicken
